client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -58,9 +58,22 @@ export const AuditAdminView: React.FC = () => {
 
   // TODO support multiple contests in batch comparison audits
   const isBatch = auditSettings.auditType === 'BATCH_COMPARISON'
-  const singleContestMenuItems = menuItems.filter(
-    i => i.title !== 'Opportunistic Contests'
-  )
+  const isBallotComparison = auditSettings.auditType === 'BALLOT_COMPARISON'
+  const filteredMenuItems = menuItems.filter(({ title }) => {
+    switch (title as ElementType<typeof setupStages>) {
+      case 'Opportunistic Contests':
+        return !isBatch
+      case 'Participants':
+        return !isBallotComparison
+      case 'Participants & Contests':
+        return !!isBallotComparison
+      default:
+        return true
+    }
+  })
+
+  if (isBallotComparison && stage === 'Participants')
+    setStage('Participants & Contests')
```

Comment:
could you solve this issue by adding another field `id` to the `ISidebarMenuItem` type? that way you could change `title` but keep `id` the same. e.g. for the participants stage, the `id` could be `Participants` but the title could be `Participants & Contests` (when it's a ballot comparison audit)